cmake/configure_files/ElementalConfig.cmake.in

```diff
@@ -1,6 +1,8 @@
 set(Elemental_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
 set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@MPI_CXX_INCLUDE_PATH@")
-set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@QD_INCLUDES@")
+IF(@QD_FOUND@)
+   set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@QD_INCLUDES@")
+ENDIF()
 set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@MPC_INCLUDES@")
 set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@MPFR_INCLUDES@")
 set(Elemental_INCLUDE_DIRS "${Elemental_INCLUDE_DIRS};@GMP_INCLUDES@")
```

Comment:
aren't they hard requirements?

Reply:
No, Elemental builds even if they aren't detected.

Reply:
ok

Reply:
Due to the fact that GMP/MPFR/MPC are only used if MPC is functioning (they are nested), it should be ready to merge if you change all three of GMP/MPFR/MPC to be included if `@MPC_FOUND@`: https://github.com/elemental/Elemental/blob/87d3b5ffd9c96834237dfc9687c5268c1a7e0cf8/cmake/external_projects/ElMath.cmake#L445

Otherwise, the case where GMP and/or MPFR were found, but not MPC, would potentially break.

Reply:
done.